packages/router/src/rsc/ClientRouter.tsx

```diff
@@ -89,5 +86,10 @@ const LocationAwareRouter = ({
     )
   }
 
-  return rscFetch('__rwjs__Routes', { location: { pathname, search } })
+  // Our types dont fully handle async components
```

Comment:
Can you mark this with a TODO please, so we don't forget to come back and clean this up?
```suggestion
  // TODO (RSC): Our types don't fully handle async components
```

Reply:
Sure